### DISCUSSION TITLE
Thank you!

### DISCUSSION THREAD

**Discussion by XavierHG** [fholger/farcry_vrmod#23] [General]
This mod renew my hope in VR gaming, I prefer classic shooter games over current VR titles.
I've always wanted to play the first FC in VR. I'm already near the end of the game and the gameplay in v0.6.0 is unbelievably great!
I want to thank everyone involved in such a professional work.

**Comment by Chattermouth** [comment]
I'll second that. 
The Prey VR mod, Unreal VR mods, and this? 
It's really impressive work, and I'm thankful they can keep an expensive headset so active.

**Comment by ianwhitedeveloper** [comment]
Thank you @fholger for creating this! This game is sooo fun to play in VR. Amazing work 🏆 💯 😎 🙏

**Comment by Kentypoo** [comment]
Fholger, you sir are a true Hero! loving the new updates and even the bhaptics support!!  ;D yay!  Think you can work on the pausing of the stick movement while using room scale at the same time? The game has such big maps and I'd much rather walk in my playspace and use manual redirected walking while using joystick than be forced to stand still if I wanna use joystick movement.. Thanks for what we have tho. You rock!

**Comment by MAXJ-afwef** [comment]
I want to say that I found this and the mod for Crysis and I have just been having an absolute blast. I think I prefer Far Cry overall because Crysis is forever buggier and basically ends when the aliens start showing up... and it has invisibility which is kind of lame... and a radar... but both games are awesome nonetheless.
If you ever felt like doing Far Fry 2 I would not mind...screw the other ones.
